### ISSUE TITLE
Constant appearance of message is irritating

### ISSUE THREAD

**Issue by anitagraham** [mbelstead/thq-connect-wp#24]
Everywhere I go in the WP site I see this message:
"You have been logged in but you are not currently connected with this organisation. Connect Now. If you are already Connected, please log out and log back in again via TidyHQ"

If I dismiss it, it returns.
It is not clear what the significance of "logged in but not connected" versus "If you are already connected" is.

I certainly understood the instruction to logout and log back in via TidyHQ to refer to going to my tidyhq page and logging out there.

I *now* think you might mean logout of wordpress and log back in, using your tidyhq username/password and clicking the tidyhq box on the WP login screen.

Which I have done. And my screen showed a tiny black wp screen atop my large white wp screen.

So to summarize:

- message is contradictory
- meaning is unclear
- action to take is unclear
- results of the action taken are unexpected
- message is persistent - even when using parts of the wp-admin not associated with tidyhq functions.

thanks, Anita
